src/frontend/magnify/src/components/rooms/RegisterRoomForm/RegisterRoomForm.tsx

```diff
@@ -62,8 +73,18 @@ const RegisterRoomForm = ({ onSuccess }: RegisterRoomFormProps) => {
     [],
   );
 
-  const handleSubmit = (values: RegisterRoomFormValues) => {
-    mutate(values);
+  const handleSubmit = (
+    values: RegisterRoomFormValues,
+    actions: FormikHelpers<RegisterRoomFormValues>,
+  ) => {
+    mutation.mutate(values, {
+      onError: (error) => {
+        const formErrors: FormErrors = error?.response?.data as FormErrors;
```

Comment:
This typing is weird. As it seems you could retrieve `undefined` but TS is not warned about that.

```suggestion
        const formErrors = error?.response?.data as Maybe<FormErrors>;
```

Reply:
Agreed